src/pages/home/sidebar/OptionRow.js

```diff
@@ -275,6 +288,14 @@ export default withLocalize(memo(OptionRow, (prevProps, nextProps) => {
         return false;
     }
 
+    if (!_.isEqual(prevProps.textStyles, nextProps.textStyles)) {
+        return false;
+    }
+
+    if (!_.isEqual(prevProps.alternateTextStyles, nextProps.alternateTextStyles)) {
+        return false;
+    }
+
```

Comment:
Not needed. :arrow_up:

Reply:
It is need to update the style.

Reply:
Based on the previous suggestion it will be not required.